### ISSUE TITLE
package installation failed

### ISSUE THREAD

**Issue by AmeerDZ1981** [GediminasB/bayesVAR_TVP#4]
I've been trying to install the bayesVAR_TVP package, but the package could not be installed, seems there is a problem with some files in the package, I always get these messages 

     Warning in get_bibentries(package = package, ..., stop_on_error = FALSE) :
     Couldn't find file REFERENCES.bib in package `bayesVAR'
       Loading required namespace: bayesVAR
       Failed with error:  'there is no package called 'bayesVAR''
       Warning in get_bibentries(package = package, ..., stop_on_error = FALSE) :
     Couldn't find file REFERENCES.bib in package `bayesVAR'
       Loading required namespace: bayesVAR
       Failed with error:  'there is no package called 'bayesVAR''
       Warning in eval(expr, env) :
       Failed to insert reference with key = KoopKorobilis2010 from package = 'bayesVAR'. 
    Possible cause --- missing REFERENCES.bib in package 'bayesVAR' or '
  
         Failed to insert reference with key = KoopKorobilis2010 from package = 'bayesVAR'. 
    Possible cause --- missing REFERENCES.bib in package 'bayesVAR' or 'bayesVAR' not installed.
       Warning in eval(expr, env) :
         Failed to insert reference with key = CC1994 from package = 'bayesVAR'. Possible cause --- 
    missing REFERENCES.bib in package 'bayesVAR' or 'bayesVAR' not installed.
       Warning in eval(expr, env) :
         Failed to insert reference with key = DK2002 from package = 'bayesVAR'. Possible cause --- 
    missing REFERENCES.bib in package 'bayesVAR' or 'bayesVAR' not installed.
       Warning in eval(expr, env) :
         Failed to insert reference with key = CogleySargent2005 from package = 'bayesVAR'. 
    Possible cause --- missing REFERENCES.bib in package 'bayesVAR' or 'bayesVAR' not installed.
       *** installing help indices
         converting help for package 'bayesVARTVP'
           finding HTML links ... done
           MinimumEntropy_weights                  html  
           bayesFAVAR_TVP                          html  
           bayesVAR_TVP                            html  
           impulse.response                        html  
           plot.beta                               html  
           predictive.density                      html  
       ** building package indices
       ** testing if installed package can be loaded
       Error: package or namespace load failed for 'bayesVARTVP' in library.dynam(lib, package, package.lib):
    DLL 'bayesVAR' not found: maybe not installed for this architecture?
       Error: loading failed
       Execution halted
       ERROR: loading failed
    -  removing 
    'C:/Users/BOUBEK~1/AppData/Local/Temp/RtmpYdejQ6/Rinst18d413135e92/bayesVARTVP'
       In R CMD INSTALL
             -----------------------------------

  
       ERROR: package installation failed

    Error: Failed to install 'bayesVARTVP' from GitHub:
      System command error

**Comment by sutong123** [comment]
Me too，Have you resolved this problem?

**Comment by AmeerDZ1981** [comment]
Yes, I did, but I don't know what happened. I just tried to install it many times until I succeeded

**Comment by sutong123** [comment]
> 
> 
> Yes, I did, but I don't know what happened. I just tried to install it many times until I succeeded

Thank you, i have also finished it by downloading the zip file in laptop and install it from local source, plessure to share with you.

**Comment by adwelch** [comment]
Hi, Sutong123: Would it be possible to give me the details of your installation process. I tried several times and but failed every time. I also got the following:
Error: Failed to install 'bayesVARTVP' from GitHub:
  (converted from warning) installation of package ‘C:/Users/Adr/AppData/Local/Temp/Rtmpy2Y1nC/file40dc30a36f0/bayesVARTVP_0.1.0.tar.gz’ had non-zero exit status

I would appreciate if you would share step by step installation process. Thanks,

**Comment by adwelch** [comment]
> > Yes, I did, but I don't know what happened. I just tried to install it many times until I succeeded
> 
> "Thank you, i have also finished it by downloading the zip file in laptop and install it from a local source, a pleasure to share with you".

Hello, I ran the following command and got the following failure output.  Any sort of help would appreciated. Thanks,
install.packages("devtools", repos = "http://cran.us.r-project.org")
library(devtools)
install_github("GediminasB/bayesVAR_TVP", force = TRUE)

........
installing to C:/Users/Adr/Documents/R/win-library/3.6/00LOCK bayesVARTVP/00new/bayesVARTVP/libs/x64
** R
** inst
** byte-compile and prepare package for lazy loading
** help
Error :  (converted from warning) Failed to insert reference with key = Robertson2005 from package = 'bayesVAR'. Possible cause --- missing REFERENCES.bib in package 'bayesVAR' or 'bayesVAR' not installed.
ERROR: installing Rd objects failed for package 'bayesVARTVP'
* removing 'C:/Users/Adr/Documents/R/win-library/3.6/bayesVARTVP'
Error: Failed to install 'bayesVARTVP' from GitHub:
  (converted from warning) installation of package ‘C:/Users/Adr/AppData/Local/Temp/RtmpOCiqnG/file34a058f04b6e/bayesVARTVP_0.1.0.tar.gz’ had non-zero exit status

**Comment by sweetdavo** [comment]
> > Yes, I did, but I don't know what happened. I just tried to install it many times until I succeeded
> 
> Thank you, i have also finished it by downloading the zip file in laptop and install it from local source, plessure to share with you.

Hi Sutong123, I have also downloaded it and installed locally. But i still get error messages. Can you Help?

install.packages("C:/Users/Desktop/FCI/bayesVAR_TVP-master.zip", repos = NULL, type = "win.binary")
require(bayesVAR_TVP-master)

c("Loading required package: -", "Loading required package: bayesVAR_TVP", "Loading required package: master")
Failed with error:  ‘'package' must be of length 1’
In addition: Warning message:
In if (!loaded) { :
  the condition has length > 1 and only the first element will be used
